hera_mc/watch_dog.py

```diff
@@ -25,19 +25,23 @@ def send_email(subject, msg, to_addr=None, from_addr='hera@lists.berkeley.edu',
 
     Returns
     -------
-    str or None
-        If skip_send, will return the composed message.
+    EmailMessage
+
     """
     import smtplib
+    from email.message import EmailMessage
     if to_addr is None:  # pragma: no cover
         to_addr = read_forward_list()
```

Comment:
Where is this function defined?

Reply:
read_forward_list is defined in watch_dog.py

Reply:
oh, I see, it's defined below where it's used. It should be moved, it's best practice to define functions before they're used.

Reply:
I've moved it up